app/api/v2/managers/schedule_api_manager.py

```diff
@@ -0,0 +1,37 @@
+from marshmallow.schema import SchemaMeta
+from typing import Any
+
+from app.api.v2.managers.operation_api_manager import OperationApiManager
+from app.objects.c_operation import OperationSchema
+from app.utility.base_world import BaseWorld
+
+
+class ScheduleApiManager(OperationApiManager):
+    def __init__(self, services):
+        super().__init__(services)
+        self.services = services
+
+    def find_and_update_object(self, ram_key: str, data: dict, search: dict = None):
+        return super(OperationApiManager, self).find_and_update_object(ram_key, data, search)
+
+    def update_object(self, obj: Any, data: dict):
+        return super(OperationApiManager, self).update_object(obj, data)
```

Comment:
I'm thinking we should support updating the inner operation as well as the scheduling info (e.g. the time)